package.json

```diff
@@ -18,7 +18,7 @@
 	"license": "AGPL-3.0-or-later",
 	"private": true,
 	"engines": {
-		"node": ">= 16"
+		"node": ">= 14"
```

Comment:
I think some changes in `trunk` were accidentally overwritten here (such as getting rid of Node 14 support)